src/scripts/migrate.py

```diff
@@ -128,8 +128,9 @@ def slurp(
 
     # Sort, add robot row, save and return
     result = pd.DataFrame(terms_to_slurp)
-    result = result.sort_values(
-        ['mondo_id', 'mondo_label', 'xref', 'xref_source', 'original_label', 'definition', 'parents'])
+    if len(result) > 0:
```

Comment:
Simple bugfix. Sorry, I try to remember to do this check where I see if the dataframe is empty before trying to sort it, but I forgot to do so here.